posix/clone-commit

```diff
@@ -12,6 +12,12 @@ if [ ! -d .git ]; then
 	set +x
 fi
 
+if [ -n "$DRONE_HTTP_PROXY_URL" ] && [ -n "$DRONE_HTTP_PROXY_PORT" ]; then
```

Comment:
is this working, i think it should be written like  "${DRONE_HTTP_PROXY_URL}"

Reply:
@devkimittal yes it work 
![image](https://github.com/wings-software/drone-git/assets/11028152/8883d24f-78e0-4a3a-a0ad-48161ae33a33)

Reply:
i see, have you tried an e2e run though? Also what about ssh clone, will it require additional changes as well?

Reply:
no, we don't support ssh via proxy for the moment. This is only http proxy. "git config --global http.proxy"